test/webserv/integration/test_http_delete.py

```
@@ -0,0 +1,171 @@
+import os
+import time
+
+import pytest
+from common_functions import read_file, send_request_and_assert_response
+from common_response import (forbidden_response, no_content_response_close,
+                             no_content_response_keep, not_allowed_response,
+                             not_found_response, timeout_response)
```

Ruff

`common_response.timeout_response` imported but unused

Remove unused import: `common_response.timeout_response`

(F401)